### PR TITLE
Fixes for rhel based pkgs

### PR DESCRIPTION
**Pull request by jjasghar**
- fixes for rhel based pkgs

Signed-off-by: JJ Asghar <jj@chef.io>

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -9,7 +9,18 @@
...
 default['rabbitmq']['deb_package_url'] = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{node['rabbitmq']['version']}/"
 
-default['rabbitmq']['rpm_package'] = "rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
+case node['platform_family']
+when 'rhel', 'fedora'
+  case node['platform_version']
+  when > '6'
+    default['rabbitmq']['rpm_package'] = "rabbitmq-server-#{node['rabbitmq']['version']}-1.el6..noarch.rpm"
+  when < '6'
+    default['rabbitmq']['rpm_package'] = "rabbitmq-server-#{node['rabbitmq']['version']}-1.el7.noarch.rpm"
```
Comment (michaelklishin) [inline attributes/default.rb]: This looks like a typo? For versions < 6 this uses the CentOS 7 package?

File: attributes/default.rb [hunk 2]
```diff
@@ -9,7 +9,18 @@
...
+  end
+when 'suse'
+  default['rabbitmq']['rpm_package'] = "rabbitmq-server-#{node['rabbitmq']['version']}-1.suse..noarch.rpm"
```
Comment (michaelklishin) [inline attributes/default.rb]: Is there an extra dot before the `noarch`?

File: attributes/default.rb (hunk 1 above)
Comment (jjasghar) [inline attributes/default.rb]: opps. thanks :)

File: attributes/default.rb (hunk 2 above)
Comment (jjasghar) [inline attributes/default.rb]: fixed

File: attributes/default.rb (hunk 1 above)
Comment (michaelklishin) [inline attributes/default.rb]: Also, we should cover the case where version equals 6? ;)

File: attributes/default.rb [hunk 3]
```diff
@@ -9,7 +9,18 @@
 default['rabbitmq']['deb_package'] = "rabbitmq-server_#{node['rabbitmq']['version']}-1_all.deb"
 default['rabbitmq']['deb_package_url'] = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{node['rabbitmq']['version']}/"
 
-default['rabbitmq']['rpm_package'] = "rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
+case node['platform_family']
+when 'rhel', 'fedora'
+  case node['platform_version']
+  when < '6'
```
Comment (michaelklishin) [inline attributes/default.rb]: I think this should be `<=` as currently version `6` will match the other branch.

Comment (jjasghar) [inline attributes/default.rb]: ah, makes sense.

**Pull request merged**